apis/extension/node.go

```diff
@@ -27,11 +27,13 @@ import (
 const (
 	// AnnotationNodeCPUTopology describes the detailed CPU topology.
 	AnnotationNodeCPUTopology = NodeDomainPrefix + "/cpu-topology"
-	// AnnotationNodeCPUAllocs describes the CPUs allocated by Koordinator LSE/LSR and K8s Guaranteed Pods.
+	// AnnotationNodeCPUAllocs describes K8s Guaranteed Pods.
```

Comment:
According to the previous design, the koordlet needs to report the CPU information of the Pod scheduled by the Koordinator. This CPU information is obtained from the cgroup. With this information, we can analyze changes and problems on the node side.

Reply:
add TODO